### DISCUSSION TITLE
Date Field BT-127-notice

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#389] [Q&A] [answered]
Hi Labs !
in notice type 4 in the form one can put the date in field (Estimated date of publication of a contract notice within this procedure- BT-127-notice ) as for example 29/03/2023, but it get a validation error for the pattern.
I guess it lacks the timezone.
This means that in java ( backend ) I have to add the timezone to the value got from front-end ?
Thanks and regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Indeed, in the XML, any date and time shall have an associated timezone. Date format shall also follow ISO 8601: YYYY-MM-DD.
